### PR TITLE
Improve feedlist class

### PR DESCRIPTION
**Pull request by dennisschagt**
Various improvements to the `FeedListFormAction` class.

### CODE REVIEW

File: src/feedlistformaction.cpp [hunk 1]
```diff
@@ -911,6 +904,13 @@ void FeedListFormAction::update_form_title(unsigned int width)
 	std::string title_format =
 		cfg->get_configvalue("feedlist-title-format");
 
+	const unsigned int unread_feeds = std::count_if(
```
Comment (Minoru) [inline src/feedlistformaction.cpp:907]: Considering this is repeated in two places already, why not move this code into a method?

Comment (dennisschagt) [reply src/feedlistformaction.cpp]: Good idea, done

**Comment by Minoru** [comment]
:+1: Will merge once CI passes.

**Comment by Minoru** [comment]
Something's up with Cirrus CI, the builds are stuck. I reported this: https://github.com/cirruslabs/cirrus-ci-docs/issues/652

**Comment by Minoru** [comment]
Apparently not all updates made it to GitHub, but it's all green on Cirrus now, so I'm merging. Thank you for this work, @dennisschagt!

**Pull request merged**